src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,137 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
```

Comment:
you don't need both head and tail, you can use on;y head as first and work with it